docs/Growing.md

```diff
@@ -100,7 +100,7 @@ Content-Length: 16
 Hello again sebastian.
 ```
 
-Traditionally all session data was stored on the server-side and only session ids were exchanged between browser and
+Traditionally all session data was stored on the server-side and only session IDs were exchanged between browser and
```

Comment:
"ids" is lower case in the original.

Reply:
Why make these arbitrary changes from the original growing guide? You didn't open a matching PR for it.

Reply:
ids is a misspelling, and I'd be happy to open a matching PR for the original guide for this and very. https://english.stackexchange.com/questions/101248/how-should-the-abbreviation-for-identifier-be-capitalized

Reply:
Please do.